### PR TITLE
add a warning

### PR DESCRIPTION
**Pull request by zcg**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Underlying tools
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**Other information:**
I see a problem in v2ex, url is [https://www.v2ex.com/t/691339#reply0](https://www.v2ex.com/t/691339#reply0) ,so I add a waring in guide.

### CODE REVIEW

**Comment by msg7086** [comment]
I don't think the message is appropriate. It's too long and is not clear enough.

Simply saying -- content of the directory would be deleted, would be enough.

File: docs/guide/cli-service.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Usage: vue-cli-service build [options] [entry|pattern]
 Options:
 
   --mode         specify env mode (default: production)
-  --dest         specify output directory (default: dist)
+  --dest         specify output directory (default: dist), warning: content of the directory would be deleted
```
Comment (kidonng) [inline docs/guide/cli-service.md:76]:
```suggestion
  --dest         specify output directory (default: dist) WARNING: all content of the directory will be deleted
```

File: docs/ru/guide/cli-service.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ npx vue-cli-service serve
 Опции:
 
   --mode             определить режим сборки (по умолчанию: production)
-  --dest             определить каталог сборки (по умолчанию: dist)
+  --dest             определить каталог сборки (по умолчанию: dist)Внимание: Содержаниекаталог будет удалён
```
Comment (kidonng) [inline docs/ru/guide/cli-service.md:75]:
```suggestion
  --dest             определить каталог сборки (по умолчанию: dist) Внимание: Содержаниекаталог будет удалён
```

File: docs/zh/guide/cli-service.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ npx vue-cli-service serve
 选项：
 
   --mode        指定环境模式 (默认值：production)
-  --dest        指定输出目录 (默认值：dist)
+  --dest        指定输出目录 (默认值：dist)，警告：构建前输出目录内容将会被删除
```
Comment (kidonng) [inline docs/zh/guide/cli-service.md:69]:
```suggestion
  --dest        指定输出目录 (默认值：dist) 警告：目录中的所有内容将被删除
```

**Pull request closed without merging**